src/Ditto.Sqs.Consumer/AppRegistry.cs

```diff
@@ -16,51 +14,35 @@ public class AppRegistry
     {
         public static ServiceCollection Register(IConfiguration configuration, ServiceCollection services)
         {
-            services.AddSingleton<IConfiguration>(configuration);
+            services.AddSingleton(configuration);
 
-            services.AddDefaultAWSOptions(configuration.GetAWSOptions());
-            services.AddAWSService<IAmazonKinesis>();
-            
             // Binds the "Settings" section from appsettings.json to AppSettings
             var settings = configuration.Bind<DittoSettings>("Settings");
             services.AddSingleton(settings);
 
-            var destinationSettings = configuration.Bind<KinesisSettings>("Kinesis");
-            services.AddSingleton(destinationSettings);
+            var consumerOptions = configuration.GetSection("Consumer").Get<ConsumerOptions>();
+            services.AddSingleton(consumerOptions);
+
+            services.AddSqsConsumers(builder =>
+                builder
+                    .FromConfiguration(configuration)
+                    .Consume("ditto",
+                        consume => consume
+                            .CreateServiceScopePerHandler()
+                            .IgnoreUnregisteredTypes()
+                            .TreatUnregisteredTypesAsHandled()
+                            .WithHandler<EventWrapper, SqsEventHandler>(nameof(EventWrapper))
+                            .WithReaderCount(1)
```

Comment:
Should we make this one configurable as well? Ultimate ability to "play" around

Reply:
I thought about that, but that will remove the sequence we have. Parallelizing here defeats the purpose of single write threads in EVS

Reply:
Interesting point indeed 🤔. How does the library handle ordering internally?